projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.stats.reap_overextend.tooltip_extended": "精疲力竭效果降低移动速度，攻击速度和方块采集速度",
+  "tetra.stats.ability_exhilaration": "振奋",
+  "tetra.stats.lunge_exhilaration.tooltip": "猛冲伤害§e+%s%%（%s）§r并显著增加攻击范围，没有击中实体则会精疲力竭",
+  "tetra.stats.lunge_exhilaration.tooltip_extended": "精疲力竭效果降低移动速度，攻击速度和方块收获速度",
+  "tetra.stats.execute_exhilaration.tooltip": "如果持有者击杀目标，则会获得基于所造成伤害的攻击加成，目标每§e%s%%§r生命值使攻击伤害§e+1§r。持续时间基于目标最大生命值，每点生命值§e+%s§r秒，最多不超过§e200§r秒",
+  "tetra.stats.slam_exhilaration.tooltip": "猛击地面时每击中1个目标伤害§e+%s%%（%s）§r。对单个目标猛击则会获得§e10§r秒的基于目标被击退距离的攻击伤害加成，在§e20§r格距离的时候获得最大加成，攻击伤害§e+%s§r",
+  "tetra.stats.puncture_exhilaration.tooltip": "如果减甲后目标的护甲值低于§e6§r点，则刺穿冷却时间重置。如果目标带有减甲效果，则使流血效果持续时间§e+%s§r秒",
+  "tetra.stats.pry_exhilaration.tooltip": "每层撬击效果使撬击伤害§e+%s%%（%s）§r",
+  "tetra.stats.overpower_exhilaration.tooltip": "如果压制击杀了目标，移除持有者身上的精疲力竭效果",
+  "tetra.stats.reap_exhilaration.tooltip": "击杀目标则收割冷却时间§e-%s%%§r\n \n§r每次击杀会为持有者提供能吸收§e%s§r点伤害的护盾，持续§e30§r秒，后续使用收割击杀将刷新此护盾",
+  "tetra.stats.ability_revenge": "复仇",
+  "tetra.stats.lunge_revenge.tooltip": "用猛冲攻击复仇目标会重置能力冷却时间",
+  "tetra.stats.lunge_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.execute_revenge": "苦痛揭示",
+  "tetra.stats.execute_revenge.tooltip": "如果持有者带有负面效果，处决伤害§e+%s%%§r",
+  "tetra.stats.slam_revenge.tooltip": "用猛击击中复仇目标使其眩晕§e%s§r秒",
+  "tetra.stats.slam_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.puncture_revenge": "逆转",
+  "tetra.stats.puncture_revenge.tooltip": "如果目标比持有者有更多的护甲值，则对目标和持有者同时施加减甲效果和流血效果",
+  "tetra.stats.pry_revenge.tooltip": "撬击对复仇目标造成伤害§e+%s%%（%s）§r并双倍降低护甲值",
+  "tetra.stats.pry_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.overpower_revenge.tooltip": "压制对复仇目标造成伤害§e+%s%%（%s）§r并且不会使持有者精疲力竭",
+  "tetra.stats.overpower_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.reap_revenge.tooltip": "收割对复仇目标造成伤害§e+%s%%（%s）§r\n \n§r每次击杀收割伤害§e+1§r，持续§e20§r秒，每击杀一个复仇目标，持续时间§e+%s§r秒",
+  "tetra.stats.reap_revenge.tooltip_extended": "最近攻击过你的实体被判定为复仇目标，持有具有复仇能力的物品并注视复仇目标时会出现一个HUD指示器，使用复仇能力会重置目标的复仇状态。所有复仇目标在30秒内未被攻击后复仇状态将重置",
+  "tetra.stats.ability_echo": "回声",
+  "tetra.stats.lunge_echo.tooltip": "在半空中时，猛冲可以额外使用§e%s§r次，以延长距离或改变方向",
+  "tetra.stats.lunge_echo.tooltip_extended": "在半空中按Shift/潜行键来向后闪避， 按空格/跳跃键（不是右键）来增加向上移动的距离",
+  "tetra.stats.execute_echo.tooltip": "3秒后再次攻击目标，造成基于其已损失生命值和负面效果的伤害",
+  "tetra.stats.slam_echo.tooltip": "3秒后对同一目标二次猛击",
+  "tetra.stats.puncture_echo.tooltip": "3秒后再次刺穿目标",
+  "tetra.stats.pry_echo.tooltip": "3秒后再次撬击目标",
+  "tetra.stats.overpower_echo.tooltip": "持有者的精疲力竭延迟§e%s§r秒生效，再次压制则进一步延迟",
+  "tetra.stats.reap_echo.tooltip": "3秒后对同一区域二次收割\n \n§r首次击杀伤害§e+1§r，持续§e30§r秒，后续击杀伤害§e+%s§r",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Trigger a secondary reap in the same area after 3 seconds
 
§rReap grants §e+1§r attack damage per kill for §e30§r seconds, attack damage gained from subsequent kills stacks up to §e+%s§r attack damage**

```suggestion
  "tetra.stats.reap_echo.tooltip": "3秒后对同一区域二次收割\n \n§r首次击杀伤害§e+1§r，持续§e30§r秒，后续击杀伤害§e+%s§r",
```